examples/spark_dataset_converter/tensorflow_converter_example.py

```diff
@@ -78,15 +78,16 @@ def run(data_dir):
     converter_test = make_spark_converter(df_test)
 
     def train_and_evaluate(_=None):
-        import tensorflow as tf
+        import tensorflow.compat.v1 as tf  # pylint: disable=import-error
 
-        with converter_train.make_tf_dataset() as dataset:
-            dataset = dataset.map(lambda x: (tf.reshape(x.features, [-1, 28, 28]), x.label))
-            model = train(dataset)
+        with tf.Graph().as_default():
```

Comment:
I tested the example without this line (no change in L83-L90) and it works well.

Reply:
Should we remove all `with tf.Graph().as_default():` from all tests and replace it with a new decorator `create_tf_graph` (without differentiating between TF2 and TF1)?

Reply:
@liangz1 This line is for TF v2 compatibility.

`tensorflow_converter_example.py` this is an example code and it shouldn't reference the decorator defined in tests code. We will mark this example as TF v1 and add example for TF v2, in separate PR.

Reply:
Sure. Did not realize this was an example...